docs/autopilot.md

```diff
@@ -1,41 +1,41 @@
-# Autopilot mode
+# Autopilot
 
-Autopilot is a feature in development that will guarantee you are always up to
-date. This initial implementation is based out of `systemd` units and disabled
-by default.
-
-The usage presented here will become part of the snappy command itself.
+*Autopilot* is a feature that will guarantee you are always up to
+date. It is enabled by default, and can be disabled via `snappy config`.
 
 ## Usage
 
-To verify if the timer is active, run
+To check whether the feature is active, run
 
-    systemctl status -l snappy-autopilot.timer
+    snappy config ubuntu-core | grep autopilot
 
-Or for more details of when it is to be triggered can be seen as well by running
+If you want to disable it run
 
-    systemctl list-timers snappy-autopilot.timer
+    echo 'config: {ubuntu-core: {autopilot: off}}' | sudo snappy config ubuntu-core -
 
-If you want to enable it run
+and you then re-enable it via
 
-    sudo systemctl enable snappy-autopilot.timer
-    sudo systemctl start snappy-autopilot.timer
+    echo 'config: {ubuntu-core: {autopilot: on}}' | sudo snappy config ubuntu-core -
 
-And consequently, do disable it run
+Every time autopilot triggers it will try to update the whole system;
+if an `ubuntu-core` update is available the system will automatically
+reboot, although a message is printed to console with instructions on
+how to abort the reboot, in case you are logged in at the time.
```

Comment:
I think we should have a config key for this option. Depending on what you deploy snappy to this is a deal-breaker in many ways. Still just a note, not related to this change.